storage/cloud-client/acl.py

```diff
@@ -0,0 +1,255 @@
+    bucket.acl.save()
+
+    print('Added user {} as an owner on bucket {}.'.format(
+        user_email, bucket_name))
+
+
+def remove_bucket_acl(bucket_name, user_email):
+    """Removes a user from the access control list of the given bucket."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # remove access for different types of entities.
+    bucket.acl.user(user_email).revoke_read()
+    bucket.acl.user(user_email).revoke_write()
+    bucket.acl.user(user_email).revoke_owner()
+    bucket.acl.save()
+
+    print('Removed user {} from bucket {}.'.format(
+        user_email, bucket_name))
+
+
+def set_bucket_default_acl(bucket_name, user_email):
+    """Adds a user as an owner in the given bucket's default object access
+    control list."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # grant access to different types of entities. You can also use
+    # `grant_read` or `grant_write` to grant different roles.
+    bucket.default_object_acl.user(user_email).grant_owner()
+    bucket.default_object_acl.save()
+
+    print('Added user {} as an owner in the default acl on bucket {}.'.format(
+        user_email, bucket_name))
+
+
+def remove_bucket_default_acl(bucket_name, user_email):
+    """Removes a user from the access control list of the given bucket's
+    default object access control list."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # remove access for different types of entities.
+    bucket.default_object_acl.user(user_email).revoke_read()
+    bucket.default_object_acl.user(user_email).revoke_write()
+    bucket.default_object_acl.user(user_email).revoke_owner()
+    bucket.default_object_acl.save()
+
+    print('Removed user {} from the default acl of bucket {}.'.format(
+        user_email, bucket_name))
+
+
+def get_blob_acl(bucket_name, blob_name):
+    """Prints out a blob's access control list."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(blob_name)
+
+    for entry in blob.acl:
+        print('{}: {}'.format(entry['role'], entry['entity']))
+
+
+def get_blob_acl_for_user(bucket_name, blob_name, user_email):
+    """Prints out a bucket's access control list for a given user."""
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(blob_name)
+
+    blob.acl.reload()
+
+    # You can also use `group`, `domain`, `all_authenticated` and `all` to
+    # get the roles for different types of entities.
+    roles = blob.acl.user(user_email).get_roles()
+
+    print(roles)
+
+
+def set_blob_acl(bucket_name, blob_name, user_email):
```

Comment:
set_blob_acl -> add_blob_owner

or something similar, to make explicit that you're adding an owner, as opposed to setting a generic acl